cmd/kured/main.go

```diff
@@ -346,8 +351,9 @@ func root(cmd *cobra.Command, args []string) {
 	log.Infof("Reboot Sentinel: %s every %v", rebootSentinel, period)
 	log.Infof("Blocking Pod Selectors: %v", podSelectors)
 	log.Infof("Reboot on: %v", window)
+	log.Infof("Force annotation cleanup after: %v", annotationTTL)
```

Comment:
could you make this more readable in the case that it isn't set?

Reply:
Added explicit message if this is disabled.